README.md

```diff
@@ -98,6 +98,7 @@ Read about guest lectures [here](course/guest-lectures.md).
 * [Hsiao-Ching (Jean) Lin](people/hsiaoching-lin.md)
 * [Huai-Che (Acsa) Lu](people/huaiche-lu.md)
 * [Inna Shteinbuk](people/inna-shteinbuk.md)
+* [Jacqueline J.ø Pang](people/jacqueline-pang.md)
```

Comment:
You should correct this typo, then amend your last commit via `git commit --amend`, then force push the change, so `git push origin master -f`, and it will just update itself here :)